lib/Partitioner/Partitioner.cpp

```diff
@@ -1005,9 +1028,30 @@ llvm::Error Partitioner::Partition(CompilationContext &cctx) {
   for (auto i = funcToBackend.begin(); i != funcToBackend.end(); ++i) {
     module_->eraseFunction(i->first);
   }
+
   auto funcList = module_->getFunctions();
+  if (logPartition) {
+    LOG(INFO) << "The number of partitions is : " << funcList.size()
+              << ", and the DAG is dumped into DAG.dot file.\n";
+    dumpDAG("DAG.dot");
+  }
+  int i = 0;
   for (Function *subF : funcList) {
     (void)subF;
+    if (logPartition) {
+      LOG(INFO) << "\t Partition " << i << ": \n"
```

Comment:
What do you think of using `VLOG(1)` for this instead of `LOG(INFO)` plus a custom option?  Using `VLOG` gives you fine-grained control over logging, so you could specify `--vmodule=Partitioner=1` to selectively enable logging for this file.

Reply:
Is it easy to specify "--vmodule=Partitioner=1" through onnxfif?  Now we used llvm option to control if we would like to dump the log. After talked with @rdzhabarov, now the log is enabled by default, and user can use "-log-partition" to disable it since using llvm parameters in onnxifi is not easy.

Reply:
Oh, enabling the log by default is fine.

For posterity `--vmodule` is a gflags command line flag.  That makes it really easy to use with Onnxifi, since no changes have to be made to Onnxifi, we'd just rely on the host program also using Gflags.

But, I guess one annoying thing is that any of our binaries that use llvm flags parsing will not have `--vmodule` available :-(

Reply:
Thanks!